src/definitions/collections/grid.less

```diff
@@ -1454,11 +1454,15 @@
         & when (@variationGridDivided) {
             /* Divided Reversed */
             .ui[class*="computer reversed"].divided.grid:not([class*="vertically divided"]) > .column:first-child,
```

Comment:
Why isnt this line also put into a separate line (because of the >) . This rule is very inconsistent to me.

Reply:
because of maxLength = 120

Prettier tries to put almost everything on 1 line and if it is too long, then it wraps it

impossible to control/setup additional wrapping rules, the only exception is to wrap after after comma `,`, I have patched it to be reasonably ok

Reply:
As stated and suggest by you below, we should set this to infinity or remove it then. As already said, causes more trouble than it helps

Reply:
done in https://github.com/fomantic/Fomantic-UI/pull/2611/commits/470cd4e7b1dbce900d5e006aa8f3c5d7bf850009